test/e2e/apimachinery/webhook.go

```diff
@@ -1191,6 +1233,28 @@ func updateConfigMap(c clientset.Interface, ns, name string, update updateConfig
 	return cm, pollErr
 }
 
+type updateCustomResourceFn func(cm *unstructured.Unstructured)
+
+func updateCustomResource(c dynamic.ResourceInterface, ns, name string, update updateCustomResourceFn) (*unstructured.Unstructured, error) {
+	var cr *unstructured.Unstructured
+	pollErr := wait.PollImmediate(2*time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
nit: I realized it's copied from existing method, but is 1 minute necessary?

Reply:
I don't remember why we made it 1 minute in the first place. I prefer to fix them in a dedicated PR if necessary.

Reply:
it tries every 2 seconds up to a minute... the minute timeout is expected to be generous